### DISCUSSION TITLE
为什么有两个官网

### DISCUSSION THREAD

**Discussion by GreatAuk** [umijs/qiankun#1597] [Q&A] [answered]
https://qiankun.umijs.org/zh
https://qiankun.umijs.org/zh/

两个地址几乎一样，就后面多了一个 `/` , `github` 上链接的是 https://qiankun.umijs.org/zh/ ，但感觉不如另一个详细

**Comment by kuitos** [comment, marked as answer]
这就是同一个，只不过兼容了不带 slash 的地址而已，你看到的不一样？

**Reply by GreatAuk** [reply]
![image](https://user-images.githubusercontent.com/20253809/127285445-2acd52b0-a3c9-4f15-8a82-6854ea7f3fb2.png)
![image](https://user-images.githubusercontent.com/20253809/127285461-50af2b75-1bd3-477e-b6bf-4557235f90ed.png)

**Reply by kuitos** [reply]
清缓存

**Reply by GreatAuk** [reply]
浏览器缓存的问题，我强制刷新了下就好了。但这个缓存怕是有点久，我前几个月好像就这样了

**Reply by kuitos** [reply]
前一个你看到的是老版本的
